taxonomies/categories.txt

```diff
@@ -92924,13 +92924,16 @@ fi:Tyrnihyytelöt
 fr:Gelées d'argousiers
 
 <en:Sweet spreads
+#en:Sweet spreads in France
```

Comment:
```suggestion
en:Sweet spreads in France
```
I would keep en: as first line. Or eventually: "en:Pâtes à tartiner"

Reply:
OK I'll do that